api/nakadi-event-bus-api.yaml

```diff
@@ -78,15 +75,21 @@ info:
 
     Notable omissions here are:
 
-    * The contract between Nakadi and clients in the context of the high-level API (i.e. the process for
-    clients to establish subscriptions).
+    For the subscription API:
 
-    * Enrichment procedure.
+    * Support for controlling, in the subscription API, specific delivery guarantees (at most once, at least once).
+    * negotiation of sessions' parameters
```

Comment:
In the previous item we started with uppercase and used a "." to finish it. We should follow the same pattern in the following items as well for consistence.